app/src/main/java/com/android/sample/ui/components/SearchBar.kt

```diff
@@ -0,0 +1,32 @@
+package com.android.sample.ui.components
+
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Search
+import androidx.compose.material3.Icon
+import androidx.compose.material3.OutlinedTextField
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.unit.dp
+import com.android.sample.resources.C.Tag.MEDIUM_PADDING
+import com.android.sample.resources.C.Tag.STANDARD_PADDING
+
+@Composable
+fun SearchBar(onValueChange: (String) -> Unit, value: String) {
+  OutlinedTextField(
+      value = value,
+      onValueChange = onValueChange,
+      label = { Text("Search") },
+      leadingIcon = { Icon(Icons.Filled.Search, contentDescription = "Search") },
+      modifier =
+          Modifier.testTag("searchBar")
+              .clip(shape = RoundedCornerShape(MEDIUM_PADDING.dp))
+              .fillMaxWidth()
+              .padding(STANDARD_PADDING.dp),
+      singleLine = true)
+}
```

Comment:
I checked out to your branch and realized that the bar was already visible when opening the overview screen, maybe we could make it visible when scrolling down so that it is hidden at first